### DISCUSSION TITLE
An option to disable 60 FPS in YT Revanced

### DISCUSSION THREAD

**Discussion by Username1-a** [ReVanced/revanced-discussions#583] [General]
It would be useful, for example, when the user gets poor cellular reception and it would just be easier on a data plan without switching to lower quality. Data Saving mode in YT settings, unfortunately, doesn't help here (it keeps 60 FPS on a videos).
It could've been achieved like it was in YT Vanced. There was a "Codec settings" menu, where you could set the codec preferences from a different phone model, which will disable 60 FPS on a videos and will just keep 30 FPS (Setting phone model to SM-T250 there, for example, disables 60 FPS).
Screenshots of this menu in Vanced:
![Screenshot_20220910-174714_YouTube~2](https://user-images.githubusercontent.com/47947180/189548686-abd79888-6fcf-4e64-9386-f2ed3a024a5b.png)
____
![Screenshot_20220911-103341_YouTube](https://user-images.githubusercontent.com/47947180/189548698-6212fa92-613c-497e-a927-73026f777afc.png)

**Comment by outusuke** [comment]
webm videos on my device lag a lot, especially if it is at 60fps, if it is possible to include a function to disable 60fps or force mp4 format I will appreciate it

**Comment by Username1-a** [comment]
Any updates on this? I saw it's closed as "resolved".

**Reply by LisoUseInAIKyrios** [reply]
It was closed because the last comment was 2 years ago and everything here is stale.

Set the default YT video quality to anything except "high quality" and you should not get 60fps.

**Reply by PatrickMitchellUI** [reply]
Many lower spec phones can handle 1080 24-30 just fine, "anything except high quality" leaves you with a max res of 480p 🤮🤮
